CHEATSHEET.md

```diff
@@ -0,0 +1,47 @@
+# Shopware PWA Setup Cheatsheet
+
+This cheatsheet will give you an overview of the Shopware PWA project and how to set it up for development.
+
+## General
+
+`showpare-pwa` is a collaborative project between Vue Storefront and Shopware with the goal of providing an alternative Storefront based on the Vue Storefront Next architecture. It uses the Shopware Sales-Channel API for communication to the Shopware backend.
+
+As both projects are still undergoing development, we decided to mitigate potential delays through an additional Shopware plugin `SwagVueStorefront` which extends the Sales-Channel API. Eventually, once the core will provide those functionalities, the plugin will be dumped and not required anymore.
+
+There will be a more strealined, interactive development setup, but for now we're focussing on progress and stability of the platform.
+
+## Installation
+
+### Prerequisites
+
+ * `shopware/platform` running within either `showpare/development` or `shopware/production`
+ * `node@^10` and `npm@^6` installed
```

Comment:
we just require yarn, npm is installed by default and its version doesn't bother us :)
```suggestion
 * `node@^10` installed
```